templates/stamp/utilities.sh

```diff
@@ -738,13 +741,15 @@ install-mailer()
     fi
 
     log "Creating new SMTP configuration template"
-    tee /etc/ssmtp/ssmtp.conf > /dev/null <<EOF
+    tee $SMTP_CONFIG_FILE > /dev/null <<EOF
```

Comment:
nit: there are ways of writing bash variables directly to a file without additional sed operations. I won't block on this, but we should start using those methods for readability and to minimize mistakes.